src/components/KeyItem/styles.js

```diff
@@ -0,0 +1,101 @@
+import styled from 'styled-components'
+import { StyledLabel } from '@components/LocaleLabel/styles'
+import { StyledTag } from '@components/Tag/styles'
+
+export const KeyWrapper = styled.div``
+
+export const KeyDetails = styled.div``
```

Comment:
Let’s remove this since we don’t use it.

Having empty selectors for `KeyWrapper` and `KeyValueColumn` is a bit weird to me, but actually cool? No need to refactor if we ever want to style them and that way we can name those instead of having `<div>`s. 👍